CODEOWNERS

```diff
@@ -51,4 +74,23 @@ platform/platform-paxweb-jettyconfig/*	@stustison
 
 platform/security/*		@stustison
 
+platform/security/security-migratable/*     @paouelle   @lambeaux
+
 platform/util/*			@stustison
+
+platform/migration/*            @paouelle   @lambeaux
+platform/platform-migratable/*  @paouelle   @lambeaux
+
+platform/platform-configuration/*       @lambeaux
+platform/platform-filter-clientinfo/*   @lambeaux
+
+
+###############################
+# ADMIN
```

Comment:
Sounds good 👍  Want me to include empty subsections as well? So it's pre-structured for later updating?